src/profile-logic/perf-script-profile-format.js

```diff
@@ -141,64 +141,78 @@ export function convertPerfScriptProfile(profile: string): Object {
     // eg, "java 12688/12764 6544038.708352: cpu-clock:"
     // eg, "V8 WorkerThread 24636/25607 [000] 94564.109216: cycles:"
     // other combinations possible
-    // pattern: thread-name-with-optional-spaces tid time: NNN cycles:XXXX:
-    // alternate pattern: thread-name-with-optional-spaces pid/tid time: NNN cycles:XXXX:
+    // pattern: thread-name-with-optional-spaces-and-numbers tid time: NNN cycles:XXXX:
+    // alternate pattern: thread-name-with-optional-spaces-and-numbers pid/tid time: NNN cycles:XXXX:
+    // eg, "FS Broker 5858  5791/5860  30171.917889:    4612247 cycles:ppp: "
     // eg, "java 25607/25608 4794564.109216: 33 cycles:uppp"
     //   (generate with "perf script -F +pid")
-    //
-    // Regexes are evil, and need descriptions (and if really in a string, would need lots of extra escapes).
-    // Match a threadname, with optional spaces, by requiring it to be followed by either a numeric tid or tid/pid
-    // '/^' + '(S.*?)(?=(?:s+(?:(?:d+)/*(?:d+)*s)))s+' +
-    // Match a numeric tid, or a numeric tid/pid
-    // '(d+)/*(d+)*s+([d.]+)' + '/';
-    const sampleStartMatch = /^(\S.*?)(?=(?:\s+(?:(?:\d+)\/*(?:\d+)*\s)))\s+(\d+)\/*(\d+)*\s+([\d.]+)/.exec(
-      sampleStartLine
+
+    // First, get the sample's time stamp and whatever comes before the timeStamp:
+    const sampleStartMatch = /^(.*) ([\d.]+):/.exec(sampleStartLine);
+    if (!sampleStartMatch) {
+      console.log('did not match', sampleStartLine);
```

Comment:
Is this `console.log` supposed to be here? It looks like it's left over from debugging. If you feel it's needed, please add a better message here.